bindings/python/iota_sdk/types/event.py

```diff
@@ -19,3 +19,22 @@ class WalletEventType(IntEnum):
     SpentOutput = 2
     TransactionInclusion = 3
     TransactionProgress = 4
```

Comment:
I know it was already like that but surely that won't work with the nested `TransactionProgressEvent`?

Reply:
not sure 🤷‍♀️ make an issue?

Reply:
Sure

Reply:
https://github.com/iotaledger/iota-sdk/issues/2121